### ISSUE TITLE
[accessability] bottom home flow

### ISSUE THREAD

**Issue by shleeable** [mastodon/mastodon-ios#240]
The bottom "home button" should mirror as a back button when you're in a child (toot thread etc).

Right now. Clicking into a thread in the middle of the feed, then click the top back button returns to the last point in the feed.
clicking the home button returns to the top of the feed while inside the child.

Please connect the top back and the bottom home to the same logic.
obviously clicking home while in the middle of the feed in home, should return to the top.

**Comment by MainasuK** [comment]
That's the iOS default navigation logic and should be a feature to quick popup to the top tab page. 
You can get the same behavior for many other apps.

**Comment by Gargron** [comment]
I agree with @shleeable on this. If I scroll down in home, and then switch to notifications tab, and then switch back, then I see home at the same scroll position. But if I navigate to a thread within home, and then click the home tab, it takes me to the top instead. That's not very useful in my opinion.

**Comment by MainasuK** [comment]
That's may cause users who skim too deep into the thread can not quickly quit. 
But I'll have a try later.
